charts/zeebe-cluster-helm/templates/statefulset.yaml

```diff
@@ -108,7 +108,7 @@ spec:
           successThreshold: {{ .Values.readinessProbe.successThreshold }}
           timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
         resources:
-          {{- toYaml .Values.resources | nindent 12 }}
```

Comment:
Hm I'm wondering why you think this is an issue. Didn't this work before for our values? :thinking: